fyndiq/fyndiq-ui
# NPM@v5

## Overview

Create `package-lock.json` from NPM@v5.

## How to test

Run `npm upgrade && npm i && npm run bootstrap` and check that there is no difference in the package-lock.json file.
